parser/goyacc/main.go

```diff
@@ -641,7 +641,8 @@ yynewstate:
 			if !ok || msg == "" {
 				msg = "syntax error"
 			}
-			yylex.Error(msg)
+			// ignore goyacc error message
```

Comment:
previously I neglect a detail that errors generated by goyacc tool are ignored.
now this is fixed @zimulala